composer.json

```diff
@@ -13,18 +13,18 @@
     ],
     "require": {
         "php": ">=8.0",
-        "symfony/framework-bundle": "^5.4|^6.0",
+        "symfony/framework-bundle": "^5.4|^6.0|^7.0",
         "symfony/mailer": "^5.4|^6.0",
```

Comment:
@kbond Is it on purpose that symfony/mailer is not updated, like the other packages?

Reply:
No, thanks for catching this. Fixed in #27.

Reply:
great, thx